contracts/MarginCalculator.sol

```diff
@@ -766,7 +760,7 @@ contract MarginCalculator is Ownable {
         // find option upper bound value
         FPI.FixedPointInt memory optionUpperBoundValue = _findUpperBoundValue(_productHash, _shortExpiryTimestamp);
         // convert spot shock value of this product to FixedPointInt (already scaled by 1e27)
-        FPI.FixedPointInt memory spotShockValue = FPI.fromScaledUint(spotShock[_productHash], SCALING_FACTOR);
+        FPI.FixedPointInt memory spotShockValue = FPI.FixedPointInt(int256(spotShock[_productHash]));
```

Comment:
change this because the function `fromScaledUint` was supposed to scale another number that's not in 1e27. 
As the comment suggested, the spotShock value is already in 1e27, so we're safe to just create this FPI struct.

Reply:
this prevent us from reading the `SCALING_FACTOR` variable